scripts/build_posix.sh

```diff
@@ -101,6 +102,11 @@ install_dependencies()
 build_dvc()
 {
 	print_info "Building dvc binary..."
+
+  if [[ "$(uname)" != 'Linux' ]]; then
+    git apply $(pwd)/scripts/darwin.patch
```

Comment:
Also, we need `PKG = "deb"` and `PKG = "rpm"` patches for `deb` and `rpm` packages that we build. See
```
if [[ "$(uname)" == 'Linux' ]]; then
	fpm_build rpm		fpm_build rpm
	fpm_build deb		fpm_build deb
else	else
	fpm_build osxpkg		fpm_build osxpkg
fi	fi
```

down below

Reply:
So basically we'll need to run `pyinstaller` twice on linux: for `deb` and `rpm` to include proper `pkg.py`

Reply:
I would do something like combining `fpm_build` and `build_dvc` into `build_pkg` and using it something like so:
```
build ()
{
    echo "PKG = \"$1\"" > dvc/build.py
    build_dvc
    fpm_build $1
}
```
and using it something like
```
if [[ "$(uname)" == 'Linux' ]]; then
	build rpm
	build deb
else
	build osxpkg
fi
```

Reply:
Patch was useful in conda's case, as there was no other good way to do the changes, but here we can actually make due without them, by simplifying it to `echo` as in the cmd above:)